bluepysnap/bbp.py

```diff
@@ -17,45 +17,21 @@
 
 """BBP cell / synapse attribute namespace."""
 
-from bluepysnap.sonata_constants import DYNAMICS_PREFIX, Edge, Node
```

Comment:
after giving this more thought, I think it might be worthwhile to just keep the BBP contants for now; it makes porting easier for internal code, but putting them into the main namespace and having more constants I think makes it more confusing.

What do you think?

Reply:
Sure, makes sense. 🤘